building/glass/branches-schema.yaml

```diff
@@ -2,43 +2,38 @@ $schema: "http://json-schema.org/draft-07/schema#"
 title: BranchesConfig
 type: object
 properties:
-  upload-targets:
-    type: array
-    items:
-      type: object
-      oneOf:
-        - properties:
-            name:
-              type: string
-            method:
-              enum: ["google-cloud-storage"]
-          required: ["name", "method"]
-          additionalProperties: false
-        - properties:
-            name:
-              type: string
-            method:
-              enum: ["rsync"]
-            user:
-              type: string
-            dir:
-              type: string
-          required: ["name", "method", "user", "dir"]
-          additionalProperties: false
   branches:
     type: array
     items:
       type: object
       properties:
         name:
           type: string
-          pattern: "^[0-9a-zA-Z_.-][0-9a-zA-Z_./-]+[0-9a-zA-Z_.-]$"
+          pattern: "[0-9a-zA-Z_-]+"
```

Comment:
Is there a reason that the anchors were removed from this expression?

Reply:
oops